components/client/src/test/integration/scala/com/hotels/styx/client/StickySessionSpec.scala

```diff
@@ -97,9 +98,13 @@ class StickySessionSpec extends FunSuite with BeforeAndAfter with ShouldMatchers
 
   def stickySessionStrategy(activeOrigins: ActiveOrigins) = new StickySessionLoadBalancingStrategy(activeOrigins, roundRobinStrategy(activeOrigins))
 
+
   test("Responds with sticky session cookie when STICKY_SESSION_ENABLED=true") {
+    val stickySessionConfig = StickySessionConfig.newStickySessionConfigBuilder().timeout(100, TimeUnit.SECONDS).build()
+
     val client = newHttpClientBuilder(backendService.id)
       .loadBalancer(stickySessionStrategy(activeOrigins(backendService)))
+      .stickySessionConfig(stickySessionConfig)
```

Comment:
It's not clear how to do this properly/generally in our case,  but we might want to test that the required 'Builder' methods (in this case, stickySessionConfig() ) have been invoked in our UnitTests instead of only during the EndToEndTests .  Ideally, we would guarantee this at compile time or at the very least at runtime.
 This is a clear short-coming of our usage of the builder pattern, we cannot guarantee that the object has been constructed with all the required parameters...